udata/core/dataservices/search.py

```diff
@@ -47,7 +52,14 @@ def parse_filters(dataservices, args):
                 api.abort(400, "Organization arg must be an identifier")
             dataservices = dataservices.filter(organization=args["organization"])
         if "is_restricted" in args:
-            dataservices = dataservices.filter(is_restricted=boolean(args["is_restricted"]))
+            dataservices = dataservices.filter(
+                access_type__in=[
+                    DATASERVICE_ACCESS_TYPE_RESTRICTED,
+                    DATASERVICE_ACCESS_TYPE_OPEN_WITH_ACCOUNT,
```

Comment:
I don't think `DATASERVICE_ACCESS_TYPE_OPEN_WITH_ACCOUNT` is restricted? (same in serialize?)

Reply:
I don't know :-) It depends the meaning of "restricted". If you prefer I can only put `DATASERVICE_ACCESS_TYPE_RESTRICTED` in this filter.

Reply:
Well I would say `restricted` == `RESTRICTED` :stuck_out_tongue:
For real though, it is our actual behaviour I think.